.circleci/config.yml

```diff
@@ -58,6 +63,15 @@ jobs:
             fi
 
 workflows:
+  version: 2
+
+  build_and_linkcheck:
```

Comment:
I think we usually use hyphens to separate multi-word job/command/workflow names in CircleCI.

Reply:
PTAL

Reply:
Nice!